Shared/mods/deathmatch/logic/lua/CLuaFunctionParser.h

```diff
@@ -19,6 +19,15 @@ class CLuaArgument;
 #include "lua/LuaBasic.h"
 #include <lua/CLuaMultiReturn.h>
 
+class ArgumentParserError
```

Comment:
I would name it `LuaFunctionError` and add a new field `m_bWarning`. So you can inherit a class from it and set whether it should produce error or warning.

Reply:
Good idea, thanks ❤